zcash_client_sqlite/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 zcash_client_backend = { workspace = true, features = ["unstable-serialization", "unstable-spanning-tree"] }
 zcash_encoding.workspace = true
 zcash_primitives.workspace = true
+orchard.workspace = true
```

Comment:
Move this under "# - Shielded protocols" next to `sapling` (because it is not a crate in this repository, and it is exposed in a public API).